os/drivers/lcd/mipi_lcd.c

```diff
@@ -348,6 +376,9 @@ FAR struct lcd_dev_s *mipi_lcdinitialize(FAR struct mipi_dsi_device *dsi, struct
 		lcddbg("ERROR: LCD Init sequence failed\n");
 	}
 	priv->config->backlight(CONFIG_LCD_MAXPOWER);
-
+#ifdef CONFIG_LCD_ST7701SN
+	lcd_buffer_1 = (uint8_t *)malloc(CONFIG_LCD_XRES * CONFIG_LCD_YRES * 2 + 1);
+	lcd_buffer_2 = (uint8_t *)malloc(CONFIG_LCD_XRES * CONFIG_LCD_YRES * 2 + 1);
+#endif
```

Comment:
There is no free. Then, do we need to get dynamic allocation instead of just global static array?

Reply:
unlike tizen, there is no exit function in driver.
What I was thinking is: 
https://github.com/Samsung/TizenRT/blob/faaec07247ec92cca066d88da53ff0dafe8cd154/os/drivers/lcd/lcd_dev.c#L153

in this lcd_dev.c , we keep the count of fd. So if this fd becomes 0, then we can free the buffer.